packages/nextcloud/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## 4.0.0
```

Comment:
Should we also add the old changelogs from the old package?

Otherwise our changelog will start at 4.0 which is odd.

Reply:
I don't think the old changelog is useful because literally everything has changed.